### DISCUSSION TITLE
Client side working, Server side not working

### DISCUSSION THREAD

**Discussion by DanielHabib** [gladly-team/next-firebase-auth#597] [Q&A]
![CleanShot 2022-12-22 at 17 26 59@2x](https://user-images.githubusercontent.com/8617779/209236291-2b3ca1cc-c373-462c-836f-104228806366.png)

I'm able to login and with firebase magic link via email and access the user object client side.

`withAuthUser` recognizes when I'm logged in correctly, but `withAuthUserTokenSSR` never thinks I'm logged in.

The cookies are being set, is there a way to debug what could be going wrong? I'm not seeing any errors on the server side

My .env.local file is being read correctly

**Comment by thanhtutzaw** [comment]
I think you also need to use withauthusertokenSSR in Home . 

![Screenshot (134)](https://user-images.githubusercontent.com/71011043/210026514-29ab1f82-6769-4480-a11b-9fecdb36c8f1.png)

**Comment by kmjennison** [comment]
Here are a few steps that can help you debug:
1. Have you set `onVerifyTokenError` and `onTokenRefreshError` in your config and checked for any helpful errors?
2. Have you set `debug: true` in your config and read through server-side and client-side debug logs for any helpful messages?
3. Have you tried the [example app](https://github.com/gladly-team/next-firebase-auth/tree/v1.x/example) with your own Firebase credentials?
4. Have you read through the [troubleshooting](https://github.com/gladly-team/next-firebase-auth/tree/v1.x#troubleshooting) Q&A?
